contentcuration/contentcuration/frontend/administration/pages/Users/EmailUsersDialog.vue

```diff
@@ -1,164 +1,156 @@
-                      </VChip>
-                    </template>
                     <span>{{ item.name }} &lt;{{ item.email }}&gt;</span>
-                  </VTooltip>
-                </template>
-              </ExpandableList>
-              <VChip
-                v-else-if="usersFilterFetchQueryParams"
-                small
-              >
-                {{ searchString }}
-              </VChip>
-            </VFlex>
-          </VLayout>
-          <VTextField
-            v-model="subject"
-            class="mt-4"
-            box
-            label="Subject line"
-            required
-            :rules="requiredRules"
-          />
-          <div class="caption grey--text">Add placeholder to message</div>
-          <div class="mb-1">
-            <VBtn
-              v-for="placeholder in placeholders"
-              :key="`placeholder-${placeholder.label}`"
-              small
-              round
-              depressed
-              color="grey lighten-4"
-              style="text-transform: none"
-              @click="addPlaceholder(placeholder.placeholder)"
+                  </KTooltip>
+                  <StudioChip
+                    :ref="`tooltip-${item.id}`"
+                    :text="item.name"
+                    :small="true"
+                    :close="recipients.length > 1"
+                    data-test="remove"
+                    @input="remove(item.id)"
+                  >
+                    <div style="max-width: 72px">
+                      <div class="text-truncate">
+                        {{ item.name }}
+                      </div>
+                    </div>
+                  </StudioChip>
+                </div>
+              </template>
+            </ExpandableList>
+            <StudioChip
+              v-else-if="usersFilterFetchQueryParams"
+              :small="true"
             >
-              {{ placeholder.label }}
-            </VBtn>
+              {{ searchString }}
+            </StudioChip>
           </div>
-          <VTextarea
-            ref="message"
-            v-model="message"
-            box
-            auto-grow
-            label="Email body"
-            required
-            :rules="requiredRules"
-          />
-        </VCardText>
-        <VCardActions data-test="buttons">
-          <VSpacer />
-          <VBtn
-            data-test="cancel"
-            flat
-            @click="cancel"
-          >
-            Cancel
-          </VBtn>
+        </div>
+        <KTextbox
+          v-model="subject"
+          class="mt-4"
+          :label="'Subject line'"
+          :required="true"
+          :invalid="errors.subject"
+          :showInvalidText="showInvalidText && errors.subject"
+          :invalidText="$tr('fieldRequiredText')"
+          :showLabel="true"
+          :appearanceOverrides="getAppearanceOverrides(errors.subject)"
+        />
+        <div class="caption grey--text">Add placeholder to message</div>
```

Comment:
Would you remove all these Vuetify helpers classes?

Relatedly, for color-related ones, such as here `grey--text`, KDS theme tokens and palette approach needs to be used instead.

Please revisit the whole file.